main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedExpirationDateValidator.java

```diff
@@ -56,8 +56,9 @@ public void validate(GtfsFeedInfo entity, NoticeContainer noticeContainer) {
                 currentDatePlusSevenDays));
         return;
       }
-      if (entity.feedEndDate().equals(currentDatePlusThirtyDays)
-          || entity.feedEndDate().isBefore(currentDatePlusThirtyDays)) {
+      if (entity.feedEndDate().compareTo(currentDatePlusThirtyDays) <= 0) {
+        // compareTo() returns 0 if both dates are equal, or a negative integer if
```

Comment:
The behavior of compareTo() is well known, so you can remove this comment.

Reply:
Done, thanks!